packages/core/admin/admin/src/content-manager/pages/EditSettingsView/components/ModalForm.js

```diff
@@ -120,10 +128,33 @@ const ModalForm = ({ onMetaChange, onSizeChange }) => {
     </GridItem>
   );
 
+  const hasTimePicker = TIME_FIELD_TYPES.includes(attributes[selectedField].type);
+
+  const timeStepField = (
+    <GridItem col={6} key="step">
+      <Select
+        value={get(fieldForm, ['metadata', 'step'], 1)}
+        name="step"
+        onChange={(value) => onMetaChange({ target: { name: 'step', value } })}
+        label={formatMessage({
+          id: getTrad('containers.SettingPage.editSettings.step.label'),
+          defaultMessage: 'Step',
+        })}
```

Comment:
Could you update the label please?

```suggestion
        label={formatMessage({
          id: getTrad('containers.SettingPage.editSettings.step.label'),
          defaultMessage: 'Interval',
        })}
```

I think to be consistent with the name of the prop, you can keep `step` as field-name and as part of the translation id.